server/database/migrations/20190529204542-create-books.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       },
       isbn: {
         allowNull: false,
-        type: Sequelize.TEXT
+        type: Sequelize.BIGINT
```

Comment:
ISBN is not BIGINT, it consists of four parts for 10 digits or 5 parts for 13 digits, separated by a dash (-)

Reply:
Biola.

Reply:
kindly revert back so biola can create a bug fix for this or you can assist him